src/Symfony/Component/Console/DependencyInjection/AddConsoleCommandPass.php

```diff
@@ -32,115 +33,164 @@ class AddConsoleCommandPass implements CompilerPassInterface
 {
     public function process(ContainerBuilder $container): void
     {
-        $commandServices = $container->findTaggedServiceIds('console.command', true);
+        $commandServices = [];
         $lazyCommandMap = [];
         $lazyCommandRefs = [];
         $serviceIds = [];
 
-        foreach ($commandServices as $id => $tags) {
+        foreach ($container->findTaggedServiceIds('console.command', true) as $id => $tags) {
+            foreach ($tags as $tag) {
+                if (isset($tag['method'])) {
+                    $commandServices[$id][$tag['method']][] = $tag;
+                } else {
+                    $commandServices[$id]['__invoke'][] = $tag;
+                }
```

Comment:
Nit: This way, it is more obvious that these two lines do the same thing with a default value.
```suggestion
                $commandServices[$id][$tag['method'] ?? '__invoke'][] = $tag;
```

Reply:
I actually had the opposite view 😅 but I'm fine with your suggestion if others prefer it

Reply:
thanks for your review 🙏

Reply:
I prefer @GromNaN's proposal ;)